core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java

```diff
@@ -24,9 +27,11 @@
     private final EncodingManager encodingManager;
     private final int motorVehicleInt;
 
+    private final double DELTA = 0.1;
+
     public DataFlagEncoderTest() {
         encoder = new DataFlagEncoder();
-        encodingManager = new EncodingManager(encoder);
+        encodingManager = new EncodingManager(Arrays.asList(encoder), 8);
```

Comment:
What I meant is that in some use cases one might not need to disable this feature in order to save bits. But something like this might be interesting for other encoded values too and a different issue could be more appropriated.